examples/react-map-gl-draw/app.js

```diff
@@ -3,24 +3,8 @@ import React, { Component } from 'react';
 import { render } from 'react-dom';
 import MapGL from 'react-map-gl';
 import { Editor, EditorModes } from 'react-map-gl-draw';
```

Comment:
What would it take to get these `EditorModes` to just be nebula.gl `EditModes`? And is this PR getting us closer to that?

Reply:
Yes. I need refactor and understand the behavior for each edit mode. Next step I will extend from `@nebula.gl/edit-modes`

Reply:
I haven't looked at the details so my comment might be off the mark, but why do we need to extend them? 

Can we improve the basic modes so that we don't need to do that to use the in `react-map-gl-draw`?

Ideally the user should be able to create a new `EditMode` and use it in either component without having to worry about extending them. 

That way the user could switch without friction between `react-map-=gl-draw` and `nebula.gl`